actions/executions_cancel.py

```diff
@@ -0,0 +1,22 @@
+from lib.action import St2BaseAction
+
+__all__ = [
+    'St2ExecutionsCancel'
+]
+
+
+class St2ExecutionsCancel(St2BaseAction):
+    def run(self, ids):
+        result = {}
+        success = True
+        for i in ids:
+            try:
+                res = self.client.executions.delete_by_id(instance_id=i)
+                if 'faultstring' in res and res['faultstring']:
```

Comment:
This could be simplified to 
```suggestion
                if res.get('faultstring'):
```

Reply:
fixed